spec/requests/api/v1/events/events_spec.rb

```diff
@@ -1,3 +1,5 @@
+require 'faker'
```

Comment:
I don't think you'll need to require this. It should already be available. Also, I noticed in your Gemfile it wasn't in any group. It should at least be in the test group for now since you're not using it elsewhere.